test/test_autograd.py

```diff
@@ -2580,6 +2581,32 @@ def gen_matrices(p):
         gradgradcheck(torch.chain_matmul, gen_matrices([3, 5, 2, 6]))
         gradgradcheck(torch.chain_matmul, gen_matrices([6, 2, 4, 8, 10]))
 
+
+    def test_profiler_tracing(self):
+        t1, t2 = torch.ones(1), torch.ones(1)
+        with torch.autograd.profiler.profile() as prof:
+            torch.add(t1, t2)
+
+        with tempfile.NamedTemporaryFile() as f:
+            prof.export_chrome_trace(f.name)
```

Comment:
To make this work on Windows, either specify `tempfile.NamedTemporaryFile(delete=False)` or pass the raw object to this function `prof.export_chrome_trace(f)`.

Reply:
The existing test at https://github.com/pytorch/pytorch/blame/master/test/test_autograd.py#L2734 simply skips Windows. 

I'd suggest passing `f` here; not deleting tempdirs results in a mess over time.

Reply:
To pass `f` we'd have to modify the API/implementation of `export_chrome_trace` quite a bit - e.g. it currently expects a str, and also opens the file - so we'd have to modify it so that if we pass in a file-object, we don't open or close. I'll send a follow up PR to do that since that change could require more discussion created -  https://github.com/pytorch/pytorch/issues/34086 to keep track of this.